code-challenges/insertion-sort/README.md

```diff
@@ -0,0 +1,19 @@
+# Challenge Summary
+Learn the insertion sort. Convert some pseudo code to code. Write tests
+
+## Challenge Description
+<!-- Description of the challenge -->
+
+## Approach & Efficiency
+The approach of the insertion sort is to pull out an element 'i' and store it as the temp. 
+Check this temp against every preceding element until you find one where either 1.) the element your checking has an index of -1 or 2.) the element youre checking is less than the temp. As you go, make sure that you 'shift' each checked element forward one space in the array. You will be creating duplicates as you go.. ie if 5 is the temp.: `[ 10,20,5,40 ]` will become `[ 10,20,20,40 ]`,  and then `[ 10, 10, 20, 40 ]` at some point in its sorting process. Eventually the duplication/shifting routine will reach the beginning of the array and the temp will be inserted in place of the 10. >> `[ 5,10,20,40 ]`
```

Comment:
If you could, on your next code challenge, walk through what is happening at each pass a little more than this. That would look something like:
- At first step this is what happens (insert visual here)